### PR TITLE
Makes the spelling of 'mjollnir' consistant and renames singularityhammer.dm to wizard_weapons.dm

### PR DESCRIPTION
**Pull request by D4C-420**
## About The Pull Request
Does as the tin says and renames all instances of 'mjolnir' to 'mjollnir' . They are TECHNICALLY both correct but because 'mjolnir' was only used for the spelling in game for the hammer itself and the icon when worn on the back I decided on two l's.  This also means I had to change the icon name for the back sprite 

If I'm missing any files with it spelt 'mjolnir' please tell me

While I was at it i also decided to change singularityhammer.dm to wizard_weapons.dm because apparently the file is supposed to be where wizard weapon are stored and having it be named after a single weapon is confusing incase people add more weapons later on. 

## Why It's Good For The Game
More consistent spelling.

Code wise it makes it more clear what the intended use for a file is
 

## Changelog
:cl:
spellcheck: hopefully changed all instances of the word 'mjolnir' to 'mjollnir'
/:cl:

### CODE REVIEW

**Comment by D4C-420** [comment]
Remove the sprite label I just changed an Icon name. Also as far as I can tell the back sprites for mjollnir went untouched during when it was resprited.

File: code/game/objects/items/wizard_weapons.dm [hunk 1]
```diff
@@ -77,12 +77,12 @@
 	return .
 
 /obj/item/mjollnir
-	name = "Mjolnir"
+	name = "Mjollnir"
```
Comment (Ghommie) [inline code/game/objects/items/wizard_weapons.dm:80]:
```suggestion
	name = "Mjöllnir"
```
If you want to accentuate it for authenticity or not. It's not mandatory, I'll merge the PR anyway.

Comment (D4C-420) [reply code/game/objects/items/wizard_weapons.dm:80]: No accentuated o.

**Comment by Ghommie** [comment]
Oh, also conflicts.

**Comment by D4C-420** [comment]
Going to just make a new branch, I seem to have messed something up

**Pull request closed without merging**